ext/mysqli/mysqli.c

```diff
@@ -568,7 +568,7 @@ PHP_MINIT_FUNCTION(mysqli)
 	mysqli_exception_class_entry = zend_register_internal_class_ex(&cex, spl_ce_RuntimeException);
 	mysqli_exception_class_entry->ce_flags |= ZEND_ACC_FINAL;
 	zend_declare_property_long(mysqli_exception_class_entry, "code", sizeof("code")-1, 0, ZEND_ACC_PROTECTED);
-	zend_declare_property_string(mysqli_exception_class_entry, "sqlstate", sizeof("sqlstate")-1, "00000", ZEND_ACC_PROTECTED);
+	zend_declare_property_string(mysqli_exception_class_entry, "sqlstate", sizeof("sqlstate")-1, "00000", ZEND_ACC_PUBLIC);
```

Comment:
The property is declared in stubs, so you should remove this one, and make the change there

Reply:
Ah, sorry, I've just realized that you're targeting PHP 8.0